distribution/changelog.txt

```diff
@@ -153,14 +153,14 @@
 
 0.4.3 (2022-12-14)
 ------------------------------------------------------------------------
+- Feature: [OpenSFX#13, #18312, objects#220] New sound effects for the Hybrid and Single Rail roller coasters.
```

Comment:
To be safe, we are okay with this? Because I hash the name part and then combine it with the number this seems a bit arbitrary.

Reply:
In this case I'd put the #18312 part first before other repositories.